src/applications/facility-locator/config.js

```diff
@@ -63,6 +58,7 @@ export const resolveParamsWithUrl = (
       }
       break;
     case 'cc_pharmacy':
+    case 'cc_provider':
```

Comment:
Not necessary because I know it's the pattern here, but if there are constants we could use for the location and service types, it could help make it a bit clearer.